recipe/meta.yaml

```diff
@@ -1,82 +1,76 @@
 {% set name = "category_encoders" %}
-{% set version = "2.2.2" %}
+{% set version = "2.5.0" %}
+{% set sha256 = "ff72d21ddf62d829f43375c67ea4934c82947f9a0c5cbf9114838624db941420" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-    sha256: 79651dc3e8cd6749bc5d404c49a79e93469016d106736326842305e076136cb0
-    folder: dist
   - url: https://github.com/scikit-learn-contrib/{{ name }}/archive/{{ version }}.tar.gz
-    sha256: b8fa5f2c2f3294ed7847e9c5486566604a6a321308dfb5377440a7ca2a9a5b1f
-    folder: src
+    sha256: {{ sha256 }}
 
 build:
-  noarch: python
+  # noarch: python
   number: 0
+  skip: true  # [py<37 or py>310]
```

Comment:
is jinja2 check 'py>310' now working?  as in the past it didn't.  Not sure if we should add here right now upper bound at all, as python 3.11 is not out, and we are not necessarily sure it will be incompatible to 3.11+.  And of course the part in the cbc already limits it pretty much to 3.7, 3.8, 3.9, and 3.10

Reply:
I normally would not put the upper bound in there but the source repository seems to be doing so. I'll remove it.

Reply:
thanks